dist/api_data.py

```diff
@@ -1229,6 +1229,16 @@ def __ge__(self, other):
         choices=['commit', 'first_commit', 'prepare', 'read']),
 ]),
 
+'WT_SESSION.query_timestamp_numeric' : Method([
```

Comment:
I don't love duplicating the information in the descriptions - could you reference the non-numeric versions doc instead? Similarly for timestamp_transaction_numeric?

Reply:
done, I have also modified wiredtiger.in to remove duplication, please check that too